channel/api/snippets/WorkspaceProvisioning/Program.cs

```diff
@@ -0,0 +1,174 @@
+using Google.Apis.Auth.OAuth2;
+using Google.Api.Gax;
+using Google.Cloud.Channel.V1;
+using Google.Type;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace CodeLab
+{
+    class Program
```

Comment:
Can you add regions so the snippets are extracted correctly in https://cloud.google.com/channel/docs/codelabs/workspace/provisioning

See https://github.com/max-mathieu/nodejs-channel/blob/master/samples/workspace-codelab.js for the new regions we have to use

Reply:
+1 On this repo we usually don't store samples that are not shown in docs.
Addign region tags will also help with knowing where to split this in individual methods and that will help greatly with testing.